src/client/common/platform/fileSystem.ts

```diff
@@ -67,15 +70,16 @@ function filterByFileType(
     if (fileType === FileType.Unknown) {
         // FileType.Unknown == 0 so we can't just use bitwise
         // operations blindly here.
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
         return files.filter(([_file, ft]) => {
             return ft === FileType.Unknown || ft === (FileType.SymbolicLink & FileType.Unknown);
         });
-    } else {
-        return files.filter(([_file, ft]) => (ft & fileType) > 0);
     }
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    return files.filter(([_file, ft]) => (ft & fileType) > 0);
 }
 
-//==========================================
+//= =========================================
```

Comment:
I think we're hitting the limits of Prettier here...

Would this work:
```suggestion
// ==========================================
```